src/App.js

```diff
@@ -1,14 +1,62 @@
 import React from 'react';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
 import AddGroceries from './AddGroceries';
 import GroceryList from './GroceryList';
 
 function App() {
   return (
-    <div className="App">
-      <AddGroceries />
-      <GroceryList />
-    </div>
+    <Router>
+      <div className="App">
+        <h1>Shopping app</h1>
+        <Switch>
+          <Route path="/grocerylist">
+            <GroceryList />
+          </Route>
+          <Route path="/addgroceries">
+            <AddGroceries />
+          </Route>
+        </Switch>
+        <nav style={{ 
+          position: 'absolute',
+          bottom: '0',
+          width: '100%'
+          }}>
+          <ul style={{
+            display: 'flex',
+            justifyContent: 'space-between',
```

Comment:
```suggestion
            justifyContent: 'space-evenly',
```

Reply:
I think this will look slightly nicer : )

Reply:
Totally agree!